### PR TITLE
Bump IgBLAST to 1.5.0

### PR DESCRIPTION
**Pull request by marcelm**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Keep also the old 1.4.0 version since the output format has changed
slightly, and programs out there that parse it may need to be updated.

### CODE REVIEW

**Comment by marcelm** [comment]
How strange! I wrote the recipe already a week ago or so, and now 1.6.0 has disappeared from the FTP server. I’ll investigate.

**Comment by marcelm** [comment]
?

**Comment by bgruening** [comment]
Upps, sorry!!!!

**Comment by marcelm** [comment]
No worries! I haven’t received a reply from the author, yet, so I think I’ll turn this into a bump to version 1.5.0 instead, but I’ll wait one or two more days.

**Comment by marcelm** [comment]
Jian Ye, who is IgBLAST’s author, replied and said that 1.6.0 had an error and will be restored soon. Version 1.5.0 already contains the features I am interested in, so I changed this PR to is a bump to version 1.5.0.

**Comment by marcelm** [comment]
@bioconda/core Could anyone have a look? The recipe is failing with the message [`wget: command not found`](https://travis-ci.org/bioconda/bioconda-recipes/jobs/163338709#L491). I’ve added `wget` to the build requirements, but it still does not work. Many other recipes I checked that use `wget` in the build script don’t even have it in their dependencies.

**Comment by bgruening** [comment]
Please use curl, this should work.

**Comment by marcelm** [comment]
Ok, I will do so. It’s still a regression, though.

**Comment by marcelm** [comment]
Hm, I actually need wget because I cannot make curl do `wget -r -nH --cut-dirs=5 ...` (recursive download).

**Comment by johanneskoester** [comment]
You should be able to set wget as a build dependency, because it is in the anaconda/default channel. You are right, it is a regression, likely caused by our recent change to use the same CentOS 6 docker image as conda-forge. But it is actually a good regression, because it forces us to put a tool that is not guaranteed to be present into the build dependencies.

**Comment by bgruening** [comment]
@johanneskoester the `wget` package is just the library not the binary.

**Comment by marcelm** [comment]
Yes, `wget` on anaconda is the [Python wget module](https://pypi.python.org/pypi/wget).

**Comment by johanneskoester** [comment]
Oh, that's a pity. So the logical next step would be to add wget to the conda-forge channel, right? Or temporarily to bioconda, and move it over at some point.

**Comment by marcelm** [comment]
Finally! This has taken a while to figure out ...

**Comment by bgruening** [comment]
@marcelm for some reason your are missing zlib as runtime dependency: `/usr/local/bin/../share/igblast/bin/igblastn: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory
`

File: recipes/igblast/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+  fn: ncbi-igblast-1.5.0-x64-linux.tar.gz  # [linux]
+  fn: ncbi-igblast-1.5.0-universal-macosx.tar.gz  # [osx]
+  md5: 984de09c68701cc75dd49c8935aa78a1  # [linux]
+  md5: 22cdb563a11cc41a753a5b4d7043e2d7  # [osx]
+  url: ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/1.5.0/ncbi-igblast-1.5.0-x64-linux.tar.gz  # [linux]
+  url: ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/1.5.0/ncbi-igblast-1.5.0-universal-macosx.tar.gz  # [osx]
+
 build:
-  number: 6
+  number: 1
```
Comment (bgruening) [inline recipes/igblast/meta.yaml]: please start next time with number: 0 if you have a new version

Comment (marcelm) [inline recipes/igblast/meta.yaml]: Sorry, my bad.

**Comment by marcelm** [comment]
Hm, the test passed, so I assumed everything was fine. But it seems that Travis did not really build the package. I’ll open a new PR to fix this.

**Pull request merged**